doc/releases/migration-guide-4.0.rst

```diff
@@ -27,6 +27,13 @@ Boards
 * :ref:`native_posix<native_posix>` has been deprecated in favour of
   :ref:`native_sim<native_sim>` (:github:`76898`).
 
+STM32
+=====
+
+* On all official STM32 boards, `west flash` selects STM32CubeProgrammer as the default west runner.
+  If you want to enforce the selection of another runner like OpenOCD or pyOCD for flashing, you should
```

Comment:
Is there a way in west to permanently select a preferred runner?

i.e. if you _really_ want to not install STM32CubeProg, can you configure west such that it always selects OpenOCD for flashing when no runner is given on the command line?

Reply:
It is possible using twister and map file.
Using `west`, I thought it was possible using [configuration options](https://docs.zephyrproject.org/latest/develop/west/build-flash-debug.html#configuration-options), which is unfortunately not the case apparently, but I think can be added.